### PR TITLE
initial W5YI ocs

### PR DESCRIPTION
**Pull request by ARBatteiger**
Initial W5YI Documentation

### CODE REVIEW

**Review by n2ygk — CHANGES_REQUESTED** [review]
Hey nice. Thanks for this.

You need to add links to make this findable to content/docs/_index.md

You can see what the site would look like by clicking on the "deploy preview" link in Checks.

File: content/docs/w5yi/_index.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+# date: 2020-09-13T15:51:12-04:00
+lastmod: 2020-09-13T15:51:12-04:00
+featured: false
+draft: false
+weight: 40
+icon: book
+icon_pack: fas
+---
+
+![W5YI](images/logo.jpg)
```
Comment (n2ygk) [inline content/docs/w5yi/_index.md:16]:
```suggestion
![W5YI](images/veclogo.jpg)
```

**Review by n2ygk — CHANGES_REQUESTED** [review]

**Comment by ARBatteiger** [comment]
for some reason the files do not look like what I have here.   They look like the original ARRL files.    I am not sure what step I missed.   I did a pull, then updated the files, the a git add, git commit, git push.    Then generated the pull request

**Comment by ARBatteiger** [comment]
looks like I need to refresh to pick up changes.   I need to fix a couple of links.  Also I added the W5YI to the main _index.md but I do not see the change in the deploy preview

**Comment by n2ygk** [comment]
> looks like I need to refresh to pick up changes.   I need to fix a couple of links.  Also I added the W5YI to the main _index.md but I do not see the change in the deploy preview
> 
> 

See failed checks. Looks like an incorrect `ref`. 

I am not a fan of this website generator....

**Comment by ARBatteiger** [comment]
Lets try again the logo image was missing

File: content/docs/_index.md.bak [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
```
Comment (n2ygk) [inline content/docs/_index.md.bak:1]: oops, you committed this backup file. Please `git rm` it and push another commit.

File: content/docs/w5yi/_index.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+
+Your team lead will be able to add you to their Exam Tools session once 
+@Experienced Support has received the file and updated your profile.
+
+
+
+## Is there a checklist of what my VE team needs to do?
+
+See [this checklist]({{< relref "checklist.md" >}})
+used by the [AA7HW VE Team] for some ideas. As with in-person sessions, there's always room for customizing how your particular team operates.
```
Comment (n2ygk) [inline content/docs/w5yi/_index.md:42]:
```suggestion
used by the [AA7HW VE Team](https://aa7hw.org/) for some ideas. As with in-person sessions, there's always room for customizing how your particular team operates.
```

File: content/docs/w5yi/greeter-script.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+3. Any distractions or anomalies could be cause to void the exam at the sole discretion of the VE Team. Anomalies include, but are not limited to:
+   1. Notable distraction by you, such as eyes not remaining on the exam and/or wandering around the room, or any noises.
+   2. Your appearing to reference or use unauthorized materials (e.g. looking at cell phone, even just the lock screen, changing windows on computer, etc.)
+   3. Another person entering the exam space (on camera, physically, or audibly).
+   4. Your getting up or leaving the room during the exam.
+4. Once you start the exam, the Volunteer Examiners will turn off their audio so as to not distract you – but we’ll be listening and watching. Please leave your audio and video on. Give us a shout if you need anything and when you are ready for grading click the "grade exam" button and let us know you are ready.
+5. Make sure to answer all the questions: Only correct answers add points; incorrect answers don't subtract, so it's best to guess rather than leave an answer blank. Good luck with the exam
+
+
+W5YI VE SCRIPT 1
```
Comment (n2ygk) [inline content/docs/w5yi/greeter-script.md:38]: Formatting is not working below. You need to replace the bullet characters with asterisks.

**Review by n2ygk — CHANGES_REQUESTED** [review]
See review comments and suggestions.

Hint (something I recently learned) about [batching accepted suggestions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes)

**Comment by ARBatteiger** [comment]
changes made, I tried to create a new pull request, but it said no differences ?

File: .gitignore.bak [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# IDEs
```
Comment (n2ygk) [inline .gitignore.bak:1]: haha oops you committed .gitignore.bak ;-)

File: content/docs/w5yi/greeter-script.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+* Welcome
+* [If the candidate has multiple devices, try to admit the computer first, and the phone next. Instruct the candidate to join
+with video only, not audio, on the phone. This is not always possible, because devices in the waiting room will not
+always be identifiable by candidate name or device type.]
+* Photo ID.
+o Is ID acceptable to all VE’s?
```
Comment (n2ygk) [inline content/docs/w5yi/greeter-script.md:44]: Presumably these `o` characters here and below are meant to be sub-bullets? You'll want to use asterisk indented.
```suggestion
  * Is ID acceptable to all VE’s?
```

File: content/docs/w5yi/checklist.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+## After session
+### Email CSCEs to each successful candidate
+* Examtools: Open each candidate.
+* Email CSCE to candidates using `04CSCE`.
+
+### Create VEC upload PDF package:
+* **New!** Examtools: Export dat and save with default name: `ExamSession_(Callsign)_YYYYMMDD_HHMM.dat`.
+* Examtools: Print Session Forms/Final Forms and download using the default name:
+  `ExamSession_{Callsign)_YYYYMMDD_HHMM.pdf`
```
Comment (n2ygk) [inline content/docs/w5yi/checklist.md:60]: Open curly, close paren:
```suggestion
  `ExamSession_(Callsign)_YYYYMMDD_HHMM.pdf`
```

**Review by n2ygk — CHANGES_REQUESTED** [review]
Very close. Just one or two minor nits and we should be good to go on the next round.

**Pull request merged**